compat/cpu.h

```diff
@@ -68,7 +72,7 @@ CpuCount(const cpu_set_t *set)
 #define CPU_AND(destset, srcset1, srcset2) CpuAnd((destset), (srcset1), (srcset2))
 /// CPU_AND replacement
 inline void
-CpuAnd(cpu_set_t *destset, const cpu_set_t *srcset1, const cpu_set_t *srcset2)
+CpuAnd(cpu_set_t *destset, cpu_set_t *srcset1, cpu_set_t *srcset2)
```

Comment:
Same here: CpuAnd() is ours.

```suggestion
CpuAnd(cpu_set_t *destset, const cpu_set_t *srcset1, const cpu_set_t *srcset2)
```